pyproject.toml

```diff
@@ -96,12 +95,13 @@ ignore = [
     'C408',  # Suggestion to use dict() over {}
     'W503',  # Starting lines with operators.
     'PT009', # use a regular assert instead of unittest-style '{assertion}'
+    'PT027', # pytest.raises() should be used instead of assertRaises()'
```

Comment:
Maybe remove the ' from the comments if not necessary?

Reply:
removed